pkg/apis/policy/v1alpha1/generated.proto

```diff
@@ -30,6 +30,15 @@ import "k8s.io/kubernetes/pkg/util/intstr/generated.proto";
 // Package-wide variables from generator "generated".
 option go_package = "v1alpha1";
 
+// Eviction evicts a pod from its node subject to certain policies and safety constraints.
```

Comment:
same comment here -- mention that it is a subresource of pod, to be used with POST?